src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -280,7 +295,7 @@ def arbeitsl_geld_2_regelsatz_m_hh_ab_2011(
     else:
         out = arbeitsl_geld_2_params["regelsatz"][2] * (
             2 + _arbeitsl_geld_2_alleinerz_mehrbedarf_m_hh
-        ) + (arbeitsl_geld_2_params["regelsatz"][3] * weitere_erwachsene)
+        ) + ((arbeitsl_geld_2_params["regelsatz"][3] + zuschlag) * weitere_erwachsene)
```

Comment:
Why also payed out to adults in the household?

Reply:
Ok, makes sense. Thanks!

It should not matter usually as Regelsatz 3 is directed to "Erwachsene Behinderte in stationären Einrichtungen"